jmix-dynattr/dynattr/src/main/java/io/jmix/dynattr/model/CategoryAttribute.java

```diff
@@ -95,6 +97,8 @@ public class CategoryAttribute implements Serializable {
     @Column(name = "NAME", nullable = false)
     private String name;
 
+    @NotNull(message = "{msg://io.jmix.dynattr.model/CategoryAttribute.code.validation.NotNull}")
```

Comment:
1. The default 'required' message is OK, no need in custom one. 
2. Get rid of unrelated change. If this field needs to be required at the DB level, then create a separate issue that will be implemented in the next minor release, since it requires a Liquibase migration script, which you've actually missed.